blog/templates/blog/post_list.html

```diff
@@ -0,0 +1,25 @@
+<html>
```

Comment:
This is not related to the password validation and shouldn't be included with the PR.